services/resource-usage-tracker/requirements/_base.in

```diff
@@ -8,13 +8,16 @@
 # intra-repo required dependencies
 --requirement ../../../packages/models-library/requirements/_base.in
 --requirement ../../../packages/settings-library/requirements/_base.in
+--requirement ../../../packages/postgres-database/requirements/_base.in
 # service-library[fastapi]
 --requirement ../../../packages/service-library/requirements/_base.in
 --requirement ../../../packages/service-library/requirements/_fastapi.in
 
 
+asyncpg # database
```

Comment:
I think the correct way to install is using sqlalchemy[postgresql_asyncpg]

Reply:
yes, removed :+1: I have all needed reqs installed through postgres-database